test/resource/meta/gen-test.json

```diff
@@ -33,6 +37,12 @@
       "path": "access.zapt",
       "name": "Access data example",
       "output": "access.out"
+    },
+    {
+      "path": "cluster_iterator.zapt",
+      "name": "Cluster iterator.",
+      "output": "cluster_out_{define:tolowercase}.out",
+      "iterator": "selectedCluster"
```

Comment:
How does one discover valid "iterator" values, @tecimovic ?  The docs talk about "availableClusters" as a value, but dbenum does not seem to match that, and nothing short of reading the code will discover "selectedCluster"....

Reply:
I need to update sdk-integration.md doc....